src/order/tests.py

```diff
@@ -100,6 +111,18 @@ def test_order_str_includes_date(self):
             delivery_date=delivery_date)
         self.assertTrue(str(delivery_date) in str(orders[0]))
 
+    def test_get_shippable_orders_active_client_only(self):
+        """
+        Should return all shippable orders for the given date.
+        A shippable order must be created in the database, and its ORDER_STATUS
+        must be 'O' (Ordered) and the client status must be client.ACTIVE
+        """
+        client = self.orders[0].client
+        client.status = client.PAUSED
+        client.save()
+        new_orders = Order.objects.get_shippable_orders()
+        self.assertNotEqual(len(self.orders), len(new_orders))
+
```

Comment:
@manumilou, nice work! I think we could add a test to check this and avoid future regressions.  so I have added and small tests.